### ISSUE TITLE
ipk.post.export_field_jpg() function  causes the AEDT Icepak solution result file to disappear.

### ISSUE THREAD

**Issue by aknsj** [ansys/pyaedt#5588]
### Before submitting the issue

- [x] I have searched among the existing issues
- [x] I am using a Python virtual environment

### Description of the bug

Hi, I am using the following function when exporting a surface temperature plot.When using this function to process the following project with only two solids, t the result file in the GUI interface disappeared.

```
def create_surface_temp_plot(plot_name,obj_list,orientation):
    if not isinstance(plot_name, str):
        raise TypeError("Plot name must be a string")
    surface_list = []
    for obj in obj_list:
        surface_list.extend(ipk.modeler.get_object_faces(obj))

    ipk.modeler.fit_all()
    ipk.post.create_fieldplot_surface(
        surface_list,
        quantityName="SurfTemperature",
        plot_name=plot_name
    )

    png_name = str(plot_name)+".png"
    ipk.post.export_field_jpg(fileName=os.path.join(project_path,png_name),plotName=plot_name,foldername = None,orientation=orientation,selections=obj_list)
    ipk.post.delete_field_plot(plot_name)
```
![Image](https://github.com/user-attachments/assets/93eed054-f569-4157-89a7-a97e8d664819)

I have carried out the following tests to reproduce the problem.
Feel free to contact me if you need additional information.
I look forward to your reply, thank you!

### Steps To Reproduce

As you can see below, when using this function to process the following project with only two solids, the following message appeared in Message Manager and then the result file in the GUI interface disappeared.
![Image](https://github.com/user-attachments/assets/ea50840b-cd8b-4d0c-a0b0-c8a6f0a25cb5)
![Image](https://github.com/user-attachments/assets/e35c32e4-611c-47dc-baa4-db0421ebaf07)
Note: IcepakDesign1.results folder still retains the results file in the .sd file, it looks like the folder and the GUI seem to have lost the link.
![Image](https://github.com/user-attachments/assets/ad9e7ff2-1275-4d00-83cd-675d0b40c4cf)
After testing, it was found that create plot in the function can run successfully.
```
ipk.post.create_fieldplot_surface(
    surface_list,
    quantityName="SurfTemperature",
    plot_name=plot_name
)
```
Then, it was found that this step when exporting plot caused the error that the result file disappeared.
```
ipk.post.export_field_jpg(fileName=os.path.join(project_path,png_name),plotName=plot_name,foldername = None,orientation=orientation,selections=obj_list)
```

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.8

### Installed packages

Package                               Version
------------------------------------- -------------------
absl-py                               2.1.0              
aiofiles                              23.1.0             
aiohttp                               3.8.4              
aiosignal                             1.3.1              
alabaster                             0.7.12             
altair                                4.2.2              
anaconda-client                       1.7.2              
anaconda-navigator                    2.0.3              
anaconda-project                      0.9.1              
annotated-types                       0.7.0              
ansys-api-fluent                      0.3.26             
ansys-api-platform-instancemanagement 1.1.0              
ansys-fluent-core                     0.17.1             
ansys-platform-instancemanagement     1.1.2              
ansys-pythonnet                       3.1.0rc3           
anyio                                 3.6.2
appdirs                               1.4.4
argh                                  0.26.2
argon2-cffi                           20.1.0
asgiref                               3.8.1
asn1crypto                            1.4.0
astroid                               2.6.6
astropy                               4.2.1
astunparse                            1.6.3
async-generator                       1.10
async-timeout                         4.0.2
atomicwrites                          1.4.0
attrs                                 20.3.0
autopep8                              1.5.5
Babel                                 2.9.0
backcall                              0.2.0
backports.functools-lru-cache         1.6.4
backports.shutil-get-terminal-size    1.0.0
backports.tempfile                    1.0
backports.weakref                     1.0.post1
backports.zoneinfo                    0.2.1
bcrypt                                3.2.0
beautifulsoup4                        4.9.3
bitarray                              1.9.2
bkcharts                              0.2
black                                 19.10b0
bleach                                3.3.0
bokeh                                 2.3.2
boto                                  2.49.0
Bottleneck                            1.3.2
brotlipy                              0.7.0
cachetools                            5.5.0
camelot-py                            0.11.0
certifi                               2023.7.22
cffi                                  1.14.5
chardet                               4.0.0
charset-normalizer                    3.0.1
click                                 8.1.7
cloudpickle                           1.6.0
clr-loader                            0.2.6
clyent                                1.2.2
colorama                              0.4.4
comtypes                              1.1.9
conda                                 4.10.1
conda-build                           3.21.4
conda-content-trust                   0+unknown
conda-package-handling                1.7.3
conda-repo-cli                        1.0.4
conda-token                           0.3.0
conda-verify                          3.4.2
contextlib2                           0.6.0.post1
contourpy                             1.1.1
cryptography                          41.0.2
cycler                                0.10.0
Cython                                0.29.23
cytoolz                               0.11.0
d2l                                   1.0.3
dask                                  2021.4.0
decorator                             5.0.6
defusedxml                            0.7.1
diff-match-patch                      20200713
distributed                           2021.4.0
Django                                4.2.16
dnspython                             2.6.1
docker                                7.1.0
docutils                              0.17
email_validator                       2.2.0
entrypoints                           0.3
et-xmlfile                            1.0.1
exceptiongroup                        1.1.2
fastapi                               0.111.0
fastapi-cli                           0.0.4
fastcache                             1.1.0
ffmpy                                 0.3.0
filelock                              3.0.12
flake8                                3.8.4
Flask                                 1.1.2
flatbuffers                           24.3.25
fonttools                             4.52.4
fpdf2                                 2.7.9
frozenlist                            1.3.3
fsspec                                0.9.0
future                                0.18.2
gast                                  0.4.0
gevent                                21.1.2
ghostscript                           0.7
glob2                                 0.7
google-auth                           2.34.0
google-auth-oauthlib                  1.0.0
google-pasta                          0.2.0
gradio                                3.19.1
greenlet                              1.0.0
grpcio                                1.65.1
grpcio-health-checking                1.62.2
h11                                   0.14.0
h5py                                  3.11.0
HeapDict                              1.0.1
helpdev                               0.7.1
html5lib                              1.1
httpcore                              0.16.3
httptools                             0.6.1
httpx                                 0.23.3
idna                                  2.10
image                                 1.5.33
imagecodecs                           2021.3.31
imageio                               2.9.0
imagesize                             1.2.0
importlib_metadata                    8.0.0
importlib_resources                   6.4.5
iniconfig                             1.1.1
intervaltree                          3.1.0
ipykernel                             5.3.4
ipython                               7.22.0
ipython-genutils                      0.2.0
ipywidgets                            7.6.3
isort                                 5.8.0
itsdangerous                          1.1.0
jason                                 0.1.7
jdcal                                 1.4.1
jedi                                  0.17.2
Jinja2                                2.11.3
joblib                                1.0.1
json5                                 0.9.5
jsonschema                            3.2.0
jupyter                               1.0.0
jupyter-client                        6.1.12
jupyter-console                       6.4.0
jupyter-core                          4.7.1
jupyter-packaging                     0.7.12
jupyter-server                        1.4.1
jupyterlab                            3.0.14
jupyterlab-pygments                   0.1.2
jupyterlab-server                     2.4.0
jupyterlab-widgets                    1.0.0
keras                                 2.13.1
keyring                               22.3.0
kiwisolver                            1.3.1
lazy-object-proxy                     1.6.0
libarchive-c                          2.9
libclang                              18.1.1
linkify-it-py                         2.0.0
llvmlite                              0.36.0
locket                                0.2.1
lxml                                  5.2.2
Markdown                              3.7
markdown-it-py                        2.2.0
MarkupSafe                            1.1.1
matplotlib                            3.7.2
matplotlib-inline                     0.1.6
mccabe                                0.6.1
mdit-py-plugins                       0.3.3
mdurl                                 0.1.2
menuinst                              1.4.16
mistune                               0.8.4
mkl-fft                               1.3.0
mkl-random                            1.2.1
mkl-service                           2.3.0
mock                                  4.0.3
more-itertools                        8.7.0
mpmath                                1.2.1
msgpack                               1.0.2
multidict                             6.0.4
multipledispatch                      0.6.0
mypy-extensions                       0.4.3
navigator-updater                     0.2.1
nbclassic                             0.2.6
nbclient                              0.5.3
nbconvert                             6.0.7
nbformat                              5.1.3
nest-asyncio                          1.5.1
networkx                              2.5
nltk                                  3.6.1
nose                                  1.3.7
notebook                              6.3.0
numba                                 0.53.1
numexpr                               2.7.3
numpy                                 1.23.5
numpydoc                              1.1.0
oauthlib                              3.2.2
olefile                               0.46
opencv-python                         4.8.0.74
openpyxl                              3.0.7
opt-einsum                            3.3.0
orjson                                3.8.6
outcome                               1.2.0
packaging                             23.2
pandas                                2.0.3
pandocfilters                         1.4.3
paramiko                              2.7.2
parso                                 0.7.0
partd                                 1.2.0
path                                  15.1.2
pathlib                               1.0.1
pathlib2                              2.3.5
pathspec                              0.7.0
patsy                                 0.5.1
pdfminer.six                          20221105
pep8                                  1.7.1
pexpect                               4.8.0
pickleshare                           0.7.5
Pillow                                8.2.0
pip                                   24.2
pkginfo                               1.7.0
platformdirs                          4.2.0
pluggy                                0.13.1
plumbum                               1.8.2
ply                                   3.11
pooch                                 1.8.1
prettytable                           3.9.0
prometheus-client                     0.10.1
prompt-toolkit                        3.0.17
protobuf                              4.25.3
psutil                                6.0.0
ptyprocess                            0.7.0
py                                    1.10.0
pyaedt                                0.9.10
pyasn1                                0.6.0
pyasn1_modules                        0.4.0
pycodestyle                           2.6.0
pycosat                               0.6.3
pycparser                             2.20
pycryptodome                          3.17
pycurl                                7.43.0.6
pydantic                              2.8.2
pydantic_core                         2.20.1
pyDes                                 2.0.1
pydocstyle                            6.0.0
pydub                                 0.25.1
pyecharts                             2.0.4
pyedb                                 0.23.0
pyerfa                                1.7.3
pyflakes                              2.2.0
Pygments                              2.18.0
pylint                                2.7.4
pyls-black                            0.4.6
pyls-spyder                           0.3.2
PyNaCl                                1.4.0
pyodbc                                4.0.30
pyOpenSSL                             20.0.1
pyparsing                             2.4.7
pypdf                                 3.14.0
PyQt5                                 5.12
PyQt5-Qt5                             5.15.2
PyQt5_sip                             4.19.19
PyQtWebEngine                         5.12
pyreadline                            2.1
pyrsistent                            0.17.3
PySide2                               5.15.2.1
PySocks                               1.7.1
pytesseract                           0.3.10
pytest                                6.2.3
python-dateutil                       2.9.0.post0
python-dotenv                         1.0.1
python-jsonrpc-server                 0.4.0
python-language-server                0.36.2
python-multipart                      0.0.9
python-pptx                           0.6.21
pytomlpp                              1.0.13
pytz                                  2021.1
pyvista                               0.44.1
PyWavelets                            1.1.1
pywin32                               306
pywin32-ctypes                        0.2.0
pywinpty                              0.5.7
PyYAML                                6.0.1
pyzmq                                 22.3.0
QDarkStyle                            2.8.1
QtAwesome                             1.0.2
qtconsole                             5.0.3
QtPy                                  1.9.0
regex                                 2021.4.4
requests                              2.31.0
requests-oauthlib                     2.0.0
rfc3986                               1.5.0
rich                                  13.7.1
rope                                  0.18.0
rpyc                                  6.0.0
rsa                                   4.9
Rtree                                 1.2.0
ruamel.yaml                           0.17.32
ruamel.yaml.clib                      0.2.7
ruamel-yaml-conda                     0.15.100
scikit-image                          0.18.1
scikit-learn                          0.24.1
scikit-rf                             1.2.0
scipy                                 1.10.1
scooby                                0.9.2
seaborn                               0.11.1
selenium                              4.11.2
Send2Trash                            1.5.0
setuptools                            52.0.0.post20210125
shellingham                           1.5.4
shiboken2                             5.15.2.1
simplegeneric                         0.8.1
simplejson                            3.19.2
singledispatch                        0.0.0
sip                                   4.19.13
six                                   1.15.0
sniffio                               1.2.0
snowballstemmer                       2.1.0
sortedcollections                     2.1.0
sortedcontainers                      2.3.0
soupsieve                             2.2.1
Sphinx                                4.0.1
sphinxcontrib-applehelp               1.0.2
sphinxcontrib-devhelp                 1.0.2
sphinxcontrib-htmlhelp                1.0.3
sphinxcontrib-jsmath                  1.0.1
sphinxcontrib-qthelp                  1.0.3
sphinxcontrib-serializinghtml         1.1.4
sphinxcontrib-websupport              1.2.4
spyder                                4.2.5
spyder-kernels                        1.10.2
SQLAlchemy                            1.4.7
sqlparse                              0.5.1
starlette                             0.37.2
statsmodels                           0.12.2
sympy                                 1.8
tables                                3.6.1
tabulate                              0.9.0
tblib                                 1.7.0
tensorboard                           2.13.0
tensorboard-data-server               0.7.2
tensorflow                            2.13.0
tensorflow-estimator                  2.13.0
tensorflow-intel                      2.13.0
tensorflow-io-gcs-filesystem          0.31.0
termcolor                             2.4.0
terminado                             0.9.4
testpath                              0.4.4
textdistance                          4.2.1
threadpoolctl                         2.1.0
three-merge                           0.1.1
tifffile                              2021.4.8
toml                                  0.10.2
toolz                                 0.11.1
torch                                 2.4.1+cu118
torchaudio                            2.4.1+cu118
torchvision                           0.19.1+cu118
tornado                               6.1
tqdm                                  4.59.0
traitlets                             5.0.5
trio                                  0.22.2
trio-websocket                        0.10.3
turtle100                             1.0.0
typed-ast                             1.4.2
typer                                 0.12.3
typing_extensions                     4.7.1
tzdata                                2024.1
uc-micro-py                           1.0.1
ujson                                 5.10.0
unicodecsv                            0.14.1
urllib3                               1.26.4
uvicorn                               0.20.0
vtk                                   9.3.0
watchdog                              1.0.2
watchfiles                            0.22.0
wcwidth                               0.2.5
webencodings                          0.5.1
websockets                            10.4
Werkzeug                              1.0.1
wheel                                 0.36.2
widgetsnbextension                    3.5.1
win-inet-pton                         1.1.0
win-unicode-console                   0.5
wincertstore                          0.2
wrapt                                 1.12.1
wsproto                               1.2.0
xlrd                                  2.0.1
XlsxWriter                            1.3.8
xlwings                               0.23.0
xlwt                                  1.3.0
xmltodict                             0.12.0
yapf                                  0.31.0
yarl                                  1.8.2
zict                                  2.0.0
zipp                                  3.4.1
zope.event                            4.5.0
zope.interface                        5.3.0

**Comment by maxcapodi78** [comment]
@aknsj I've tested with latest version of PyAEDT and the solution is not lost. I've checked and it seems you are using a very old version of pyaedt (0.9.10). Maybe you need to update to latest version of PyAEDT. Next monday 0.13.0 will be released and you could probably benefit of it

**Comment by Samuelopez-ansys** [comment]
Also I recall there were some problems in old AEDT releases, please test 2024R2.

**Comment by aknsj** [comment]
Thank you for your replies. I really appreciate it as I guess it might be Christmas holiday time now.
I have updated the pyaedt version to the latest version 0.13.0.
As you can see below, the Ansys Electronics Desktop version is 2024R2.1 and the python version is 3.8.8. However, based on the previously tested project, the same error still occurs. So it seems that this issue cannot be solved by upgrading the pyaedt version.
Also, I found that this issue occurs in some projects and not in some. Maybe I need some time to collect test results from different projects in the same environment and sort out the projects before contacting you.

![Image](https://github.com/user-attachments/assets/a0ef4b0a-8eb9-4b09-b21a-c8c63de3d381)

**Comment by aknsj** [comment]
Thank you for your replies. I really appreciate it as I guess it might be Christmas holiday now.
I have updated the pyaedt version to the latest version 0.13.0.
As you can see below, the Ansys Electronics Desktop version is 2024R2.1 and the python version is 3.8.8. However, based on the previously tested project, the same error still occurs. So it seems that this issue cannot be solved by upgrading the pyaedt version.
Also, I found that this issue occurs in some projects and not in some. Maybe I need some time to collect test results from different projects in the same environment and sort out the projects before contacting you.

![Image](https://github.com/user-attachments/assets/a0ef4b0a-8eb9-4b09-b21a-c8c63de3d381)
